src/stats/src/GaussianJointPdf.C

```diff
@@ -139,17 +139,23 @@ GaussianJointPdf<V,M>::actualValue(
 
   queso_require_equal_to_msg(domainVector.sizeLocal(), this->m_domainSet.vectorSpace().dimLocal(), "invalid input");
 
-  queso_require_msg(!(gradVector || hessianMatrix || hessianEffect), "incomplete code for gradVector, hessianMatrix and hessianEffect calculations");
+  queso_require_msg(!(hessianMatrix || hessianEffect), "incomplete code for gradVector, hessianMatrix and hessianEffect calculations");
 
   double returnValue = 0.;
 
-  if (this->m_domainSet.contains(domainVector) == false) { // prudenci 2011-Oct-04
+  if (this->m_domainSet.contains(domainVector) == false) {
```

Comment:
No, it's testing whether or not `domainVector`, the point we're evaluating the pdf at, is actually inside the domain or not.  The user may have constructed a bounded domain.  In this case, the pdf is zero (no probability).